sys/arm64/arm64/copyinout.S

```diff
@@ -60,11 +56,12 @@ __FBSDID("$FreeBSD$");
 #if __has_feature(capabilities)
 	.arch_extension	c64
 #endif
+/* XXX-AM: ifdef out the hybrid entrypoint in purecap? */
```

Comment:
It's currently structured so you can unconditionally use copyio_c64_fault in the places where you're using C64 in a hybrid kernel, even for non-CHERI kernels, so I don't think you want to. I would probably not add the `.arch_extension` above for purecap though, but it's harmless.